schema/me/conversation/invoice.js

```diff
@@ -0,0 +1,44 @@
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLEnumType } from "graphql"
+import { amount } from "../../fields/money"
+
+const InvoiceState = new GraphQLEnumType({
+  name: "InvoiceState",
+  values: {
+    UNPAID: {
+      value: "unpaid",
+    },
+    PAID: {
+      value: "paid",
+    },
+    VOID: {
+      value: "void",
+    },
+    REFUNDED: {
+      value: "refunded",
+    },
+  },
+})
+
+export const InvoiceType = new GraphQLObjectType({
+  name: "Invoice",
+  desciption: "Fields of an invoice (currently from Lewitt)",
+  fields: {
+    id: {
+      description: "Impulse's invoice id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    lewitt_invoice_id: {
+      description: "Lewitt's invoice id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    payment_url: {
+      description: "Link to public checkout page.",
+      type: GraphQLString,
+    },
+    state: {
+      description: "Current state of invoice.",
+      type: InvoiceState,
+    },
```

Comment:
In practice, probably not.

But there isn't any database constraint or anything _actually_ enforcing that on the Lewitt side, so I wouldn't want to cast it as such here. If we do that, we'll get an ugly GraphQL error, rather than a more graceful fallback.